hack/install-cert-manager.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
where are we using this script?

Reply:
To make cert manager install in makefile more resilient. Didn't notice I forgot to change that in the makefile. Did that now.